VOM_Fortran/VOM-code/modules.f90

```diff
@@ -372,42 +375,58 @@ module vegmod
       REAL*8  :: lambdat_d              ! Target dE/dA for calculating gstomt (mol/mol)
       REAL*8  :: lambdag_d              ! Target dE/dA for calculating gstomg (mol/mol)
       REAL*8  :: gstomt                 ! Tree stomatal conductance (mol/m2/s)
-      REAL*8  :: gstomg(3,3,3)          ! Grass stomatal conductance (mol/m2/s)
-
-      REAL*8  :: rlt_h(3,3)             ! Tree leaf respiration for different values of Jmax (rlt_h(2) is actual value) (mol/h)
+      REAL*8  :: gstomts                ! Shaded tree stomatal conductance (mol/m2/s)      
+      REAL*8  :: gstomg(3,3,3)            ! Grass stomatal conductance (mol/m2/s)
```

Comment:
Is this then sunlit stomatal conductance? Would be good to specify for all variables where there is a shaded counterpart now.

Reply:
Yes, it is, but in case the VOM is run without the shaded and sunlit leaves (it is an option), it is this gstom that does everything. I'll change it to !(Sunlit) grass stomatal conductance

Reply:
That would be great, also for the other variables that refer to sunlit leaves.